### PR TITLE
Use `parquet_2gb_sorted` for gpuCI query tests

### PR DESCRIPTION
**Pull request by sarahyurick**
https://github.com/rapidsai/ops/issues/2672

### CODE REVIEW

File: tests/unit/test_queries.py [hunk 1]
```diff
@@ -34,6 +33,7 @@
     69,
     70,
     72,
+    75,
```
Comment (sarahyurick) [inline tests/unit/test_queries.py:36]: Needs https://github.com/dask-contrib/dask-sql/pull/1153

File: tests/unit/test_queries.py [hunk 2]
```diff
@@ -42,6 +42,7 @@
     89,
     92,
     94,
+    95,
```
Comment (sarahyurick) [inline tests/unit/test_queries.py:45]: Needs https://github.com/dask-contrib/dask-sql/pull/1146

**Review by sarahyurick — COMMENTED** [review]
Not sure what to make of the failures with queries 53 and 63: https://gpuci.gpuopenanalytics.com/job/dask/job/dask-sql/job/prb/job/dask-sql-prb/2794/CUDA_VER=11.5,LINUX_VER=ubuntu18.04,PYTHON_VER=3.9,RAPIDS_VER=23.06/testReport/junit/tests.unit/test_queries/test_gpu_query_q53_sql_/

I don't see them locally.

**Comment by ayushdg** [comment]
Looks like https://github.com/dask-contrib/dask-sql/pull/1154#pullrequestreview-1440617650 seems to be coming from an updated Numba version to 0.57 with window functions.
Need to investigate further but should be good to xfail until then

**Comment by ayushdg** [comment]
Opened https://github.com/rapidsai/cudf/issues/13426 to track

**Review by ayushdg — APPROVED** [review]

**Comment by ayushdg** [comment]
Looks like the cuml failures should be fixed via https://github.com/rapidsai/cuml/pull/5434. In the interest of unblocking CI, we should be good to ignore those failures until cuml import errors are resolved.

**Pull request merged**